File: Leetcode/3024.go
package Leetcode

import "sort"

func triangleType(nums []int) string {
	sort.Ints(nums)
	if nums[0]+nums[1] <= nums[2] {
		return "none"
	} else if nums[0] == nums[2] {
		return "equilateral"
	} else if nums[0] == nums[1] || nums[1] == nums[2] {
		return "isosceles"
	} else {
		return "scalene"
	}

}

File: Leetcode/3024.py
from typing import List


class Solution:
    def triangleType(self, nums: List[int]) -> str:
        arr = nums
        arr.sort()
        if arr[0] + arr[1] <= arr[2]:
            return "none"
        if arr[0] == arr[1] == arr[2]:
            return "equilateral"
        if arr[0] == arr[1] or arr[1] == arr[2]:
            return "isosceles"
        return "scalene"
